src/resolvers/Mutation/removeOrganization.ts

```diff
@@ -27,7 +27,7 @@ export const removeOrganization: MutationResolvers["removeOrganization"] =
   async (_parent, args, context) => {
     const currentUser = await User.findOne({
```

Comment:
Change the `findOne` to `exists` function as you aren't accessing any other properties on the `User` object in your resolver.

Reply:
ok

Reply:
here we can't use exists() because we are also checking for super admin. Meaning, we have to pass the user info in it

Reply:
Got it. You can resolve this conversation.